returnn/util/debug.py

```diff
@@ -182,6 +182,13 @@ def excepthook(exc_type, exc_obj, exc_tb):
 
     sys.excepthook = excepthook
 
+    def threading_excepthook(args, /):
+        """threading-specific excepthook"""
+        better_exchook(args["exc_type"], args["exc_value"], args["exc_traceback"], file=log.v1 or sys.stdout)
```

Comment:
Did you test this? The [doc](https://docs.python.org/3/library/threading.html#threading.excepthook) says that `exc_type` and co are *attributes* of `args`.
```suggestion
        better_exchook(args.exc_type, args.exc_value, args.exc_traceback, file=log.v1 or sys.stdout)
```

If you did not test this, can you maybe test this? Maybe even write a simple test case, where some thread is spawn, raises some random exception.